Alternative Visualizations/Ethnocentrism - Alternative Visualization.nlogo

```diff
@@ -363,7 +363,7 @@ mutation-rate
 mutation-rate
 0.0
 1.0
-0.0050
```

Comment:
I think that's right. The resaving happens in Java 6, while new development happens in Java 8. scala.js also tends to treat trailing zeros differently from the JVM, so if we ever start resaving for NetLogo Web we will probably see more changes there as well.